deployment/vm_operation.sh

```diff
@@ -67,6 +67,58 @@ power_action() #id_string action(on/off/reset)
     done
 }
 
+snapshot_action()
+{
+    action=$1
+    duration=$2
+    shift 2
+    for id in $@
+    do
+        snapshot_count=`vim-cmd vmsvc/snapshot.get $id | grep "Snapshot Id"| wc -l`
+        vm_name=`vm_getname $id`
+
+        if [ $snapshot_count -gt 1 ]; then
```

Comment:
When the vmslave ansible updated, the vmslave will be re-configure, and the snapshot will be re-created.
so is it reasonable to only allow only one snapshot in total ?
when we revert to snapshot, Can we always revert to latest/last snapshot ?
```
   last_snapshot_id=$(  vim-cmd  vmsvc/snapshot.get ${vmid}|   grep "Snapshot Id"|  tail -1|awk '{print $NF}')
   last_snapshot_name=$(vim-cmd  vmsvc/snapshot.get ${vmid}| grep "Snapshot Name"|  tail -1|awk '{print $NF}')
```